indexer/locals.tf

```diff
@@ -5,9 +5,24 @@ locals {
   rds_port      = 5432
 }
 
+locals {
+  // Needed so that there are no circular dependencies for all resources are created per service names.
+  service_names = {
```

Comment:
Can't we just use a list here?

Reply:
No, it'll make it unclear when we're using it in the `services` local variable below (e.g. service_names[0] is much less clear than service_names["ender"]).